app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -65,6 +65,10 @@ open class HomeActivity : AppCompatActivity() {
 
     override fun onResume() {
         super.onResume()
+        // There is no session, or it has timed out; we should pop everything to home
+        if (components.core.sessionStorage.current() == null) {
```

Comment:
Should we check that we're not in a custom tab? Otherwise, this change is much-needed. Thanks!

Reply:
I tested this with custom tabs and it seems to work, it will pop but only in the main Fenix app (f the tab has timed out) and not the custom tab browsing